### PR TITLE
Catch errors on the server object

### PR DESCRIPTION
**Pull request by clarenced**
Catch the errors on the server object by displaying the message error.

### CODE REVIEW

File: web_server.js [hunk 1]
```diff
@@ -56,3 +56,7 @@ io.on('connection', function (socket) {
 server.listen(process.env.PORT || 80, () => {
 	log.info(`Server is listening on port: ${server.address().port}`);
 });
+
+server.on('error', error => {
+	log.error(`Error while starting the server : ${error.message}`);
```
Comment (mbrandau) [inline web_server.js]: Please log the whole error and not just the message and remove the space in front of the colon.

Comment (mbrandau) [reply web_server.js]: Another thing: please move the error listener above the `server.listen` call.

Comment (clarenced) [reply web_server.js]: OK

**Comment by mbrandau** [comment]
Fixes #28

**Pull request merged**